tests/syntax_error/missing_global_keyword.jou

```diff
@@ -1 +1 @@
-foo: int  # Error: missing 'global' keyword when defining global variable
+foo: int  # Error: missing 'global' or 'const' keyword when defining global variable
```

Comment:
Can you delete this test while you're here? It is the same exact test as `missing_global_keyword_type.jou`.